rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -71,5 +78,13 @@ class QuotingPanel {
     document.getElementById('SubmitQuote').addEventListener('submit', (event) => { event.preventDefault() });
   }
 
+  getData() {
+    return {
+      polygons: this.polygons,
```

Comment:
IMHO this should spit out the "clean" (non-Google) polygons instead of leaving that up to the `Quote` to [figure out how to deal with Google Maps objects](https://github.com/rosadiaz/snow_queen_v2/pull/7/files#diff-9067261253110cf18cd55210cbc898f8R47). E.g.
```javascript
getData() {
  return {
    polygons:  this.plainPolygons(), 
    geocodedAddress: this.geocodedAddress || "",
    // ...
  }
}

plainPolygons() {
  return this.polygons.map(this.plainPolygon);
}

plainPolygon(polygon) {
  return polygon.getPath().getArray().map(vertex => { return vertex.toJSON() })
}
```

Reply:
you're hard to please. I though to put it here because it is not the map massaging the data but the quote form before it saves to the dtabase

Reply:
to be refactored later